content/lang/fr/index.md

```diff
@@ -0,0 +1,233 @@
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/productive.svg" alt="productive">
+
+    <h2>Une Productivité Instantanée</h2>
+    
+    <p>
+        Ne sacrifiez plus votre productivité pour concevoir une application légère. Preact vous permet d'être productif dès le début et apporte même quelques fonctionnalités bonus:
+    </p>
+    
+    <ul>
+        <li>`props`, `state` et `context` sont passées à `render()`</li>
+        <li>Utilisez les attributs HTML standards comme `class` et `for`</li>
+        <li>Fonctionne avec React DevTools sans aucune modification</li>
+    </ul>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/compatible.svg" alt="compatible">
+
+    <h2>Un Ecosystème Compatible</h2>
+    
+    <p>
+        Les Composants DOM Virtuel permettent de partager des éléments réutilisables - tout est réutilisable des boutons jusqu'aux data providers.
+        Le design de Preact vous permet de réutiliser sans aucune modification des milliers de composants de l'écosystème React.
+    </p>
+    
+    <p>
+        Ajouter un simple alias <a href="/guide/switching-to-preact#how-to-alias-preact-compat">preact-compat</a> à votre bundle fournit une couche de compatibilité qui vous permettra d'utiliser les composants React les plus complexes dans votre application.
+    </p>
+</section>
+
+
+<section class="home-top">
+    <h1>Exemple</h1>
+</section>
+
+
+<section class="home-split">
+    <div>
+        <h2>Composant Liste de Tâches</h2>
+        <pre><code class="lang-js">
+export default class TodoList extends Component {
+    state = { todos: [], text: '' };
+    setText = e =&gt; {
+        this.setState({ text: e.target.value });
+    };
+    addTodo = () =&gt; {
+        let { todos, text } = this.state;
+        todos = todos.concat({ text });
+        this.setState({ todos, text: '' });
+    };
+    render({ }, { todos, text }) {
+        return (
+            &lt;form onSubmit={this.addTodo} action="javascript:"&gt;
+                &lt;input value={text} onInput={this.setText} /&gt;
+                &lt;button type="submit"&gt;Ajouter&lt;/button&gt;
+                &lt;ul&gt;
+                    { todos.map( todo =&gt; (
+                        &lt;li&gt;{todo.text}&lt;/li&gt;
+                    )) }
+                &lt;/ul&gt;
+            &lt;/form&gt;
+        );
+    }
+}
+        </code></pre>
+    </div>
+    
+    <div>
+        <h2>Démonstration</h2>
+        <pre repl="false"><code class="lang-js">
+import TodoList from './todo-list';
+
+render(&lt;TodoList /&gt;, document.body);
+        </code></pre>
+        <div class="home-demo">
+            <todo-list></todo-list>
+        </div>
+    </div>
+</section>
+
+
+<section class="home-split">
+    <div>
+        <h2>Télécharger les Etoiles Github</h2>
```

Comment:
`Récupérer des Stars Github`. Why not translate stars, but `télécharger` (download) seems strange to me in this context. Maybe make the sentence a bit more complex: `Récupérer le nombre de Stars Github d’un repository`.
**`É`**`toiles`